src/CodiceFiscaleServiceProvider.php

```diff
@@ -75,13 +75,17 @@ private function packagePath($path)
 
     public function registerFakerProvider(): void
     {
+        if(!class_exists(Factory::class)){
+            return;
+        }
+        
         $this->app->singleton(Generator::class, function () {
             $faker = Factory::create();
             $faker->addProvider(new CodiceFiscaleFakerProvider($faker));
 
             return $faker;
         });
-
+        
         fake()->addProvider(app(CodiceFiscaleFakerProvider::class));
```

Comment:
Do you think it could be a good idea to wrap this line within a...
```
if (function_exists('fake') { ... }
```
?

Reply:
Laravel checks if factory exists before defining fake() so I guess it is fine to have the same check as well

Just let me know your opinion for this

Reply:
It is not a life changer, but maybe is safer to add the check. Would you proceed with that? Thanks once again.

Reply:
done ;)